content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
 # Roles and Responsibilities
 
 ## Team Organization
+Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the quiz at any given time.
 
 ### Quizzers
 
 ### Captains
+- Scorekeepers must be notified which quizzer is captain and co-captain before the quizzers take their seats to begin the quiz.
+- The team captain and co-captain may change during a given competition but not during a quiz.
+- Only the team captain or co-captain may challenge a decision or respond to a challenge.  If both the captain and co-captain have errored out, another quizzer can be designated by the coach to respond to or initiate challenges.   
 
 ### Coaches
 
 ## Quiz Officials
+- No comment other than “correct” or “incorrect” need be announced by the quiz officials in making a decision.  
+    - However, at the discretion of the answer judge and quizmaster or at the request of the captain, additional information may be given:
+        - If the additional information regarding the ruling is provided at the discretion of the AJ/QM, and not at the request of the captain, this is not considered an explanation of a ruling (which would invalidate a Challenge or Protest).
+        - If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling, and invalidates a Challenge or Protest. Before replying to a request for explanation, the QM must ask all teams if they wish to Challenge.
 
 ### Quizmasters
+The Quizmaster should: 
+- Be thoroughly acquainted with all rules, procedures, and material of that quiz year.
+- Be impartial and render decisions consistently.
+- See that all officials are briefed regarding rules and procedures.
+- Read at a normal conversation rate.  Hesitating or reading too quickly must be avoided since either may cause inaccurate quizzing.
+    - Read all questions at the same rate of speed with the ability to stop reading the question at the exact moment a jump is made.
+- Keep the quiz moving at a consistent pace.  A quiz should not be rushed, but it is important to maintain the quickest pace which does not interfere with accuracy.
+- Be open to a challenge and be willing to evaluate whatever the captains or co- captains feel is in error.
+- Facilitate a positive, uplifting environment throughout the quiz.  Quizmasters should speak clearly and be enthusiastic about quizzing. To this end, quizzers should be called by name, the score should be announced often (when a scoreboard is not present), and bonuses and quiz-outs should be promptly recognized.
+
+The Quizmaster must:
+- Identify the question type of each question and begin every question with, “question number_____, question”.
```

Comment:
I moved this from SHOULD to MUST. Thoughts?  

Accepting that challenge on this during GWI was so dumb. And adding it to should is also dumb. It's required.

Reply:
Agreed.